Local ACF JSON and Child Theme (#29)
CodeForAfrica/HURUmap-Dashboard: README.md

4. Activate/Install other plugins you will need for your development (i.e `custom posts`, `advance custom fields`, `ACF to REST API`)
5. All done! You can now add content, use and customize plugins.

### Child Theme

To create a child theme with the hurumap as the parent theme;

>  - Create a folder in the  `wp-content/themes` folder  with the desired name eg (hurumap-child)

Reviewer: ```suggestion
 - Create a folder in the  `wp-content/themes` folder  with the desired name e.g. (hurumap-outbreak-africa)
```
outbreak.AFRICA is a more established name at the moment (has it's own domain even) so lets use that as an example.

 - Add styles.css as a subfile which will contain all of the CSS rules and declarations that control the look of your theme. Your stylesheet must contain the below required header comment at the very top of the file. This tells WordPress basic info about the theme, including the fact that it is a child theme with a particular parent.
```
/*

> Theme Name: HURUmap Promise Tracker

Reviewer: ```suggestion
Theme Name: HURUmap outbreak.AFRICA
```

Theme URI: https://github.com/CodeForAfrica/HURUmap-Dashboard
Template: hurumap
Author: CodeForAfrica
Author URI: https://github.com/CodeForAfrica/HURUmap-Dashboard

> Description: HURUmap theme

Reviewer: ```suggestion
Description: HURUmap Dashboard theme for outbreak.AFRICA
```

> Tags: hurumap

Reviewer: ```suggestion
Tags: hurumap, outbreak, covid19
```
I'm assume we can have a list here

> Version: 2019.1.1614073053

Reviewer: I don't even want to ask where we go this version number from!
